SECURITY.md

```diff
@@ -1,5 +1,7 @@
 # Caliptra Project Security Incident Response
 
+***Please note that this policy document is a work in progress. This will continue to be updated and finalized as Caliptra reaches the 1.0 milestone.***
```

Comment:
- Recommend removing `Please`
- `This will...`-->`We will continue to update this policy document until Caliptra reaches the 1.0 milestone.`

Reply:
done